msteams-platform/sbs-file-handling-in-bot.yml

```diff
@@ -488,7 +488,7 @@ items:
        :::image type="content" source="../msteams-platform/assets/images/teams-file-upload-bot/sbs-4.png" alt-text="received":::
       
         > [!NOTE]
-        >  After the bot is installed in the personal scope, the user enters into an one on one chat conversation with the bot.
+        >  After the bot is installed in the personal scope, the user enters into a one on one chat conversation with the bot.
```

Comment:
Just say it as one-on-one chat. With hyphenation.

Reply:
Hyphen is added as advised.